process/smartContract/process.go

```diff
@@ -242,6 +242,11 @@ func (sc *scProcessor) DeploySmartContract(
 		return err
 	}
 
+	vmInput.ContractCode, err = hex.DecodeString(string(vmInput.ContractCode))
```

Comment:
put these lines inside sc.createVMDeployInput

Reply:
Can not do that as the transformation should be done only when deploy is done. When the SC is called, the func names should be plain text as there are more easily read and debugged. Can be changed in the future if we really want.

Reply:
Done